utils/contributors.js

```diff
@@ -5,8 +5,32 @@ const CONTRIBUTERS = [
         image: "/images/cops.jpg",
         description: "Hello everyone, this is the first card. Go ahead, add your own!",
         github: "https://github.com/COPS-IITBHU",
+    },{
+        handle_name: 'SaiRev0',
+        name: "Saiyam Jain",
+        image: "/images/saiyam.png",
```

Comment:
You have not added any image file. If you prefer to to add any images then remove the image path line from code.